tasks/patch_wordpress.js

```diff
@@ -280,4 +285,76 @@ module.exports = function(grunt) {
 
 	})
 
+	grunt.registerTask( 'upload_patch_to_wordpress_trac', 'Upload the current diff of your develop-wordpress directory to WordPress trac', function( ticketNumber ) {
```

Comment:
this plugin is used by core WordPress, but also bbpress and buddypress (and for all I know, more places as well, but I doubt it). How about `upload_patch_trac` as the name?  Yes, WordPress is in the name of the project, but it basically means "wordpress style projects" for us.